pkg/rancher-desktop/components/MountTypeSelector.vue

```diff
@@ -110,11 +108,7 @@ export default (Vue as VueConstructor<Vue & VuexBindings>).extend({
       this.$emit('update', property, value);
     },
     disabledVirtIoFsTooltip(disabled: boolean): { content: string } | Record<string, never> {
-      let tooltip = {};
-
-      if (disabled) {
-        tooltip = { content: this.t(`prefs.onlyWithVZ_${ this.arch }`) };
-      }
+      let tooltip = { content: this.t('prefs.virtiofsDisabled')};
```

Comment:
It might have been easier to just duplicate the text for both options instead of changing the code, but this is fine too (and we'd have need to change it to show the correct text on Linux anyway).